xrpl4j-integration-tests/src/test/java/org/xrpl/xrpl4j/tests/AbstractIT.java

```diff
@@ -87,7 +88,12 @@ protected Wallet createRandomAccount() {
     // Create the account
     SeedWalletGenerationResult seedResult = walletFactory.randomWallet(true);
     final Wallet wallet = seedResult.wallet();
-    logger.info("Generated testnet wallet with XAddress={} (Classic={})", wallet.xAddress(), wallet.classicAddress());
+    String network = System.getProperty("useTestnet") != null ? "testnet" :
+      (System.getProperty("useDevnet") != null ? "devnet" : "local rippled");
+    logger.info(
+      "Generated {} wallet with XAddress={} (Classic={})",
+      network, wallet.xAddress(), wallet.classicAddress()
+    );
```

Comment:
What if we added a couple of methods in `XrplEnvironment`, one that returns the name (like `"local rippled"` or `"testnet"` or `"devnet"`), and one that returns the explorer URL to use. Each implementation of `XrplEnvironment` will easily know that information, and then you don't have to do all this switching on `System.getProperty`

Reply:
Agree - though I was thinking that would be implemented via https://github.com/XRPLF/xrpl4j/issues/266

Reply:
Part of this has been accomplished by https://github.com/XRPLF/xrpl4j/pull/178/commits/d917186a4017d75c83260468d21cc3506590444c where it adds a Custom Environment. Should be pretty quick to create these envs once NFT is merged

Reply:
Resolving this for now. We'll address it when we merge in NFTs.